src/api/window/window.ts

```diff
@@ -342,6 +342,11 @@ interface WindowMovementOptions {
  * @property { number } y Y coordinate of area's starting point
  */
 
+/**
+ * @typedef { Object } WindowMovementOptions
```

Comment:
Better use 'object' instead of 'Object'.

Reply:
Will do, I copied this from the `Area` typedef up above, so I'll change that as well